stage1/usr_from_coreos/manifest.d/systemd

```diff
@@ -6,7 +6,6 @@ bin/systemd-ask-password
 bin/systemd-cat
 bin/systemd-cgls
 bin/systemd-cgtop
-bin/systemd-coredumpctl
```

Comment:
To extend what I wrote - currently build system is also able to validate the list. It bails out when something on the list couldn't be copied.